kernelci.org/content/en/blog/posts/2024/strategic-updates/index.md

```diff
@@ -0,0 +1,48 @@
+
+## Enabling new infrastructure to run tests
+
+Our legacy system has shown its age and it has been failing to meet the growing testing needs of the community. It is a quite limited, unstable python2 project that focuses on embedded hardware. We continue on the journey to put in place our new infrastructure, so we can finally move away from the legacy KernelCI system.
+
+We bring good news! The new core service for running tests is already in place, but still going through a stabilization phase. So, the team is ramping up the number of tests slowly to deal with issues that arise especially when it comes to the quality of the testing KernelCI provides. We do not want to repeat past mistakes with results that can't really be trusted, so our focus right now is on quality rather than quantity. The team is iterating quickly on that process to enable open, wide adoption in the coming months.
+
+In our new KernelCI infrastructure, we already have support to run tests in labs (through [LAVA](https://www.lavasoftware.org/) only at the moment), Docker containers, Kubernetes and natively. Adding new labs or test environments should be relatively straightforward. Then, as we add more test environments, we are also laying down the foundation for integration with other CI systems across the community so we can share kernel builds and offer test environments.
+
+As we shared [before](https://kernelci.org/blog/posts/2023/api-early-access/), the new infrastructure expose an API for users to create accounts and drive test themselves. At the moment, we are focusing on enabling our own pipeline there, so we can shutdown the legacy system. But anyone is welcome to request a user account and try it out.
+
+Another initiative from the KernelCI community is the [GitLab CI support in the mainline](https://lore.kernel.org/lkml/20240228225527.1052240-1-helen.koike@collabora.com/) kernel. Here, the goal is to offer maintainers a CI environment that they can manage themselves. With time, KernelCI API can be leveraged to provide a backend for builds and test runs for repositories using GitLab CI.
+
+## Trusting tests results and reports
+
+On one end, we are stabilizing our new infrastructure to run the tests. On the other end, we are looking into improving the quality of the reports KernelCI sends out, so maintainers and developers can actually trust them. Given the huge amount of data coming out of test systems and bots today, we have to invest in improving the delivery of the results, or else, KernelCI will be contributing to increasing the maintainer burnout rather than helping solve it. That means improving the quality and confidence of the data, so maintainers and developers only receive reports packed with relevant information and no noise or false-positives.
```

Comment:
```suggestion
On one end, we are stabilizing our new infrastructure to run the tests. On the other end, we are looking into improving the quality of the reports KernelCI sends out, so maintainers and developers can actually trust them. Given the huge amount of data coming out of test systems and bots today, we must invest in improving the delivery of the results, or else, KernelCI will be contributing to increasing the maintainer burnout rather than helping solve it. That means improving the quality and confidence of the data, so maintainers and developers only receive reports packed with relevant information and no noise or false-positives.
```